js/remove.ts

```diff
@@ -3,55 +3,46 @@ import * as msg from "gen/msg_generated";
 import * as flatbuffers from "./flatbuffers";
 import * as dispatch from "./dispatch";
 
-/** Removes the named file or (empty) directory synchronously. Would throw
- * error if permission denied, not found, or directory not empty.
- *
- *       import { removeSync } from "deno";
- *       removeSync("/path/to/empty_dir/or/file");
- */
-export function removeSync(path: string): void {
-  dispatch.sendSync(...req(path, false));
+export interface RemoveOption {
+  recursive?: boolean;
 }
```

Comment:
I'm not a huge fan of these Options interfaces with just a single member... 
But I think it makes sense to mirror the interface in `deno.mkdir(d, { recursive: true })`

cc @piscisaureus here is another example of a function which will never have more than one option.

Reply:
Nope! An option could be whether to remove readonly files also (not a thing on linux, but kinda important on windows and a thing on os x also)

Reply:
A good argument in favor of using options objects, even when they have a single member, is simply how confusing [boolean traps](https://ariya.io/2011/08/hall-of-api-shame-boolean-trap) are.

Reply:
FYI we recently added this to the style guide https://github.com/denoland/deno_std/blob/bef7ba14303997a05cdd1ea0d28ffefeee75c993/README.md#exported-functions-max-2-args-put-the-rest-into-an-options-object